demos/launch/simulation.launch.xml

```diff
@@ -3,6 +3,9 @@
 <launch>
   <arg name="map_name" help="Name of the rmf_demos map to simulate" />
   <arg name="use_ignition" default="0" />
+  <arg name="use_crowdsim" default="1" />
+
+  <let name="menge_resource_path" value=""/>
```

Comment:
I would probably disable it by default, we probably don't want to change the behavior so drastically for users (a simple update and rebuild and suddenly there is people everywhere getting in the way of robots and glitching through doors!).

Also since the `menge_resource_path` variable has the same value for both ignition and gazebo you can just assign it here?